pyanaconda/modules/common/structures/group.py

```diff
@@ -65,5 +81,29 @@ def gid(self) -> Int:
         return self._gid
 
     @gid.setter
-    def gid(self, gid: Int):
+    def gid(self, gid: UInt32):
         self._gid = gid
+
+    def get_gid(self):
+        """Return a value suitable for use in Kickstart.
```

Comment:
Please, don't use the name `kickstart` in DBus structures. These methods are useful for other use cases, that's why they are implemented here and not in the module.